index.bs

```diff
@@ -161,14 +161,9 @@ This proposal outlines an asynchronous API using Promises/async functions for th
             [[Service-Workers|service worker]] contexts
     * [[#intro-monitor|monitor]] [=script-visible=] cookies for changes using `CookieChangeEvent`
         * ... in long-running script contexts (e.g. `document`)
-        * ... after registration during the `InstallEvent`
-            in ephemeral [[Service-Workers|service worker]] contexts
         * ... again including for script-supplied in-scope request paths
```

Comment:
Remove "again including" from the next bullet point?

I think that was supposed to bridge to the bullet point we're deleting.